File: esercizi_java/NetBeansProjects/ciaone/src/gestione/Noleggio.java
package gestione;

/**
 *
 * @author tobiasacchetto
 */
public class Noleggio {
    protected veicolo v;
    protected int n, codice;
    
    public Noleggio(veicolo v,int Noleggio){
    this.v = v;
    n = Noleggio;
    codice = v.cod;
    }
    
    public int returnNoleggio(){
        return n;
    }
    
    public String toString(){
    return "("+v.targa+","+n+")";
    }
    
}

File: esercizi_python/compito_python/Noleggio.py

class Noleggio:
    def __init__(self, v, Noleggio):
        self.v = v
        self.n = Noleggio
        self.codice = v.cod

    def toString(self):
        return "("+ self.v.targa+","+str(self.n)+")"